src/assessment.js

```diff
@@ -31,7 +38,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [fire];
```

Comment:
Yes, things in the `fire` scope have access to `laserQuantity`, but so do things in the `deathStar` scope.